presto-product-tests-launcher/src/main/resources/docker/presto-product-tests/run-presto.sh

```diff
@@ -10,6 +10,11 @@ fi
 set -x
 
 tar xf /docker/presto-server.tar.gz -C /docker
+
+for entrypoint in /docker/entrypoint.d/*; do
```

Comment:
this is a misnomer -- entrypoint should be an entrypoint (sth run before the the command)

maybe we change the the dir to `/docker/presto-init.d/` ?